.github/workflows/linux-artifacts.yml

```diff
@@ -0,0 +1,83 @@
+name: Build Linux artifacts
+on:
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - 'docs/**'
+      - 'images/docs-builder/**'
+      - '*.adoc'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'images/docs-builder/**'
+      - '*.adoc'
+env:
+  IMAGE_NAME_E2E: crc-e2e
+  IMAGE_NAME_INTEGRATION: crc-integration
+jobs:
+  build-installer:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ['1.21']
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Build Linux installer
+        run: |
+          make out/linux-arm64/crc
+      - name: Upload linux binary
+        uses: actions/upload-artifact@v4
+        with:
+          name: linux-binary
+          path: "out/linux-arm64/crc"
+  build-qe:
```

Comment:
I would prefer not having this specific flow. Currently we have the [Build Windows artifacts] where windows bits are built and also the qe images, may we can rename it to just builder and add there the linux builds.

Take into account if Linux bits are already build / check in some other flow remove it from there. The goal should be build only once.... otherwise if in future the build process requires something else on any element we need to be aware they are being built in several places with several goals.